lib/pacemaker/pcmk_sched_location.c

```diff
@@ -197,19 +197,19 @@ generate_location_rule(pe_resource_t *rsc, xmlNode *rule_xml,
 }
 
 static void
-unpack_rsc_location(xmlNode *xml_obj, pe_resource_t *rsc_lh, const char *role,
+unpack_rsc_location(xmlNode *xml_obj, pe_resource_t *rsc, const char *role,
                     const char *score, pe_working_set_t *data_set,
```

Comment:
Do you have a future patch lined up to remove the data_set argument here?

Reply:
Yep :)